CMakeLists.txt

```diff
@@ -0,0 +1,98 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
Add standard copyright up front, along with appropriate documentation.

Update docs/guide/installing to say how to use this with MSVC.